### PR TITLE
fix Chinese transitions and change the ranked tier medal

### PR DESCRIPTION
**Pull request by GravityMsc**
fix Chinese transitions according valve's update
change the position and size of the ranked tier medal
fix the position of avatar to be center

### CODE REVIEW

**Comment by GravityMsc** [comment]
Emmm... We should add some rank_tier description in 'lang'. such as Divine ELite, Divine Top.
Now I use 'No. xxx' to instead.

File: src/components/Player/Header/PlayerHeader.jsx [hunk 1]
```diff
@@ -107,78 +150,64 @@ const getRegistrationBadge = registered => registered && (
...
-    } else if (star >= 5) {
-      correctStar = 5;
+    if (rankTier === 76) {
+      let iconPath = '';
+      if (leaderboardRank > 100) {
+        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7a.png';
+      } else if (leaderboardRank > 10) {
+        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7b.png';
+      } else {
+        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7c.png';
```
Comment (howardchung) [inline src/components/Player/Header/PlayerHeader.jsx]: won't this cause 7c to display if there is no leaderboard rank?  Should 7a be the default?

Comment (GravityMsc) [reply src/components/Player/Header/PlayerHeader.jsx]: That's my fault, 7a should be the default.
But I think if rank tier is 76, leaderboard rank will be exist certainly. So I wrote `if(rankTier === 76)` in line 157

Comment (GravityMsc) [reply src/components/Player/Header/PlayerHeader.jsx]: I change some code 
```
if (rankTier === 76 && leaderboardRank) {
      let iconPath = '/assets/images/dota2/rank_icons/rank_icon_7a.png'; // Divine Elite
      if (leaderboardRank <= 10) {
        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7c.png'; // Divine Top 10
      } else if (leaderboardRank <= 100) {
        iconPath = '/assets/images/dota2/rank_icons/rank_icon_7b.png'; // Divine Top 100
      }
      ...
}
```

File: src/components/Player/Header/PlayerHeader.jsx [hunk 2]
```diff
@@ -107,78 +150,62 @@ const getRegistrationBadge = registered => registered && (
...
-    const intRankTier = parseInt(rankTier, 10);
-    const iconPath = `/assets/images/dota2/rank_icons/rank_icon_${parseInt(intRankTier / 10, 10)}.png`;
-    const star = parseInt(intRankTier % 10, 10);
-    let correctStar = 0;
-    if (star <= 0) {
-      correctStar = 0;
-    } else if (star >= 5) {
-      correctStar = 5;
+    if (rankTier === 76 && leaderboardRank) {
+      let iconPath = '/assets/images/dota2/rank_icons/rank_icon_7a.png'; // Divine Elite
```
Comment (howardchung) [inline src/components/Player/Header/PlayerHeader.jsx]: curious, is "Divine Elite" an official Valve specification?

Comment (GravityMsc) [reply src/components/Player/Header/PlayerHeader.jsx]: image
I found this image in a dota2 website of our country, not sure about it. Maybe in the US, you can easily find it.

Comment (howardchung) [reply src/components/Player/Header/PlayerHeader.jsx]: do you have a link to that site?

Comment (GravityMsc) [reply src/components/Player/Header/PlayerHeader.jsx]: the origin link which I found is [this](https://weibo.com/3992841761/Fyl23p21n?type=comment#_rnd1512637808952)
Weibo is our country's twitter, but I still don't find a official annoucement about ranked medals

Comment (GravityMsc) [reply src/components/Player/Header/PlayerHeader.jsx]: And according to some comments, OMG is using the turbo mode

**Pull request merged**